modules/ROOT/partials/nav.adoc

```diff
@@ -0,0 +1,20 @@
+.xref:index.adoc[Documentation Overview]
+* ownCloud Server Releases
+** xref:server_releases.adoc[Server Releases]
+** xref:server_release_notes.adoc[Server Release Notes]
+* xref:client_releases.adoc[ownCloud Client Releases]
+* xref:how_to_contribute.adoc[How to Contribute]
+
+// note, atm we cant include an existing component navigation via eg
+// include::{latest-server-version}@server:ROOT:nav$partials/nav-server.adoc
+// for details about how to setup the nav link properly the see:
+// https://antora.zulipchat.com/#narrow/stream/282400-users/topic/Include.20partial.20with.20ROOT.20module.20errors 
+// BUT: the content gets presented, the links do not work - therefore not used
+// a soulution needs to be considered if the current layout should be changed
+
+.ownCloud Documentations
```

Comment:
```suggestion
.ownCloud Documentation
```

IMO Documentation is a group word already and covers the general concept of one and/or more documents.